### PR TITLE
Automate post release process

### PR DESCRIPTION
**Pull request by ikhoon**
Motivation:

Automate the remaining release process to relieve the effort of the devs.

Modifications:

- Add `post-release.ts` that does some tasks after releasing a new version:
  - Close milestone.
  - Write release notes.
  - Create a pull request to Central Dogma for updating Armeria version.
- Make `post-release.ts` triggered after finishing the `publish` task.

Result:

Enhanced release workflows.

### CODE REVIEW

**Comment by ikhoon** [comment]
The workflow was verified on my fork repository. 
https://github.com/ikhoon/armeria/actions/runs/5355814594/jobs/9714547927
https://github.com/ikhoon/armeria/releases/tag/armeria-1.24.11-test

File: .github/actions/post-release.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+/**
+ * Perform post-release process.
+ * - Close milestone.
+ * - Write release notes.
+ * - Create a pull request to Central Dogma for updating Armeria version.
+ */
+async function main(): Promise<void> {
+  const tag: string = process.argv.slice(2).shift();
+  const releaseVersion = tag.replace("armeria-", "");
+  const owner = 'ikhoon';
```
Comment (yeomiji) [inline .github/actions/post-release.ts:17]:
```suggestion
  const owner = 'line';
```

File: core/src/test/java/com/linecorp/armeria/client/limit/ConcurrencyLimitWithRetryTest.java [hunk 2]
```diff
@@ -72,6 +75,18 @@ void exceptionThrownUpstream(ExchangeType exchangeType) {
         }
     }
 
+    @Test
+    void test() {
```
Comment (yeomiji) [inline core/src/test/java/com/linecorp/armeria/client/limit/ConcurrencyLimitWithRetryTest.java:79]: cruft?

File: .github/actions/post-release.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  await octokit.rest.actions.createWorkflowDispatch({
+    owner: owner,
+    repo: 'centraldogma',
+    workflow_id: 'update-armeria-version.yml',
+    ref: 'main',
+    inputs: {
+      armeria_version: releaseVersion
+    },
+  })
+  console.log("⛓️ https://github.com/line/centraldogma/actions/workflows/update-armeria-version.yml has been triggered")
```
Comment (ikhoon) [inline .github/actions/post-release.ts:55]:
```suggestion
  console.log("⛓️ https://github.com/line/centraldogma/actions/workflows/update-armeria-version.yml has been triggered.")
```

**Review by trustin — APPROVED** [review]
Lovely!

**Review by jrhee17 — APPROVED** [review]
Sorry about the late check! Thanks @ikhoon 🙇 👍 🚀

**Pull request merged**